bundles/io/org.eclipse.smarthome.io.transport.mdns/ESH-INF/config/transport-mdns.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0
+	http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="system:transport-mdns">
+		<parameter name="useOnlyOneAddress" type="boolean">
+			<label>Use only one address</label>
+			<description>Use only one IP address per interface and family</description>
+			<limitToOptions>true</limitToOptions>
```

Comment:
AFAIK you do not need this for booleans and according to https://www.eclipse.org/smarthome/documentation/development/bindings/xml-reference.html#xml-structure-for-configuration-descriptions `true` is the default anyway. Can you remove it please?